src/Codeception/Lib/Parser.php

```diff
@@ -119,9 +119,6 @@ public static function validate($file)
 
     public static function load($file)
     {
-        if (PHP_MAJOR_VERSION < 7) {
-            self::validate($file);
-        }
```

Comment:
I think that validate method can be deleted now, the only other place where it is used is a [Cept Loader](https://github.com/Codeception/Codeception/blob/c26038596f7f35cc684b380f6e3fe186331e7820/src/Codeception/Test/Loader/Cept.php#L18), but Cept Format [handles ParseError](https://github.com/Codeception/Codeception/blob/c26038596f7f35cc684b380f6e3fe186331e7820/src/Codeception/Test/Cept.php#L41-L44).

Reply:
thanks